static/playHLSVideo.js

```diff
@@ -61,6 +71,24 @@
                 });
             }
 
+            function addQualitySelector(videoElement, hlsInstance, availableLevels) {
+                var qualitySelector = document.createElement('select');
+                availableLevels.forEach(function (level, index) {
+                    var option = document.createElement('option');
+                    option.value = index.toString();
+                    option.text = level.height + 'p';
```

Comment:
```suggestion
                    var bitrate = (level.bitrate / 1_000_000).toFixed(2);
                    option.text = level.height + 'p (' + bitrate + ' mbps)';
```

Reply:
Optionally, divide by 1000 and drop the round (leave it as an integer - kbps is specific enough).

![image](https://github.com/redlib-org/redlib/assets/69441971/8264f4b6-6101-455c-86c6-8cb791df6c12)

I think this is the better way to go now tbh - Reddit brutally compresses video and mbps is far less useful IMO